vfs-azure/src/main/java/com/sludev/commons/vfs2/provider/azure/AzFileName.java

```diff
@@ -0,0 +1,64 @@
+package com.sludev.commons.vfs2.provider.azure;
+
+import org.apache.commons.vfs2.FileName;
+import org.apache.commons.vfs2.FileType;
+import org.apache.commons.vfs2.provider.AbstractFileName;
+
+
+public class AzFileName extends AbstractFileName {
+
+    private String account;
+    private String container;
+
+
+    public AzFileName(String scheme, String account, String container, String path, FileType type) {
+
+        super(scheme, path, type);
+
+        this.account = account;
+        this.container = container;
+    }
+
+
+    public String getAccount() {
+        return account;
+    }
+
+
+    public String getContainer() {
+        return container;
+    }
+
+
+    /**
+     * Factory method for creating name instances.
+     *
+     * @param absolutePath The absolute path.
+     * @param fileType     The FileType.
+     * @return The FileName.
+     */
+    @Override public FileName createName(String absolutePath, FileType fileType) {
```

Comment:
I believe normal convention (and ours) is to put annotations like `@Override` on their own line (so `public FileName createName(...` goes on the next line)